feat(button): created button with the Link to coupons

melissasidisid opened the pull request:
Created a button component along with the test

Zino0031 commented on src/pages/index.jsx:
The update to the index.jsx file is important for this feature but not for now. However, I'd like to reiterate that no changes should be made to the index.js file to prevent conflicts with other contributors and to maintain existing functionality.

Zino0031 commented on src/pages/index.jsx:
```diff
@@ -4,13 +4,17 @@ import { serverSideTranslations } from "next-i18next/serverSideTranslations";
 import * as React from "react";
 
 import Layout from "@/layout/Layout";
+import Button from "@/components/Button/Button";
```
Remove this line

Zino0031 commented on src/pages/index.jsx:
```diff
@@ -4,13 +4,17 @@ import { serverSideTranslations } from "next-i18next/serverSideTranslations";
 
     return (
         <Layout>
+            <Button />
```
Also remove this one

Zino0031 reviewed (changes requested):
The changes in this pull request introduce a new button component and make some updates to the homepage component. Overall, it's a valuable addition. However, I want to stress that no changes should be made to the index.js file. This is to prevent conflicts with other contributors and to maintain existing functionality.

DigitalDeveloperAM commented on src/components/Button/Button.jsx:
```diff
@@ -0,0 +1,14 @@
+function Button() {
+    return (
+        <Link href='/coupons'>
+            <button class='py-2 px-4 bg-green-500 text-white font-semibold shadow-md hover:bg-green-700 focus:outline-none focus:ring-2 focus:ring-green-400 focus:ring-opacity-75 rounded-full'>
```
Change the button color (#8CC540)

DigitalDeveloperAM reviewed (changes requested):
I hope you're doing well. I've had a chance to review the recent changes you made to the color button in our project. Overall, it's looking great, but there are a few adjustments that need to be made to button color that match our  Figma design.

** The color of the button need to match this #8CC540 .

Keep up the Great work, and I appreciate your hard work and dedication to the project.

melissasidisid commented:
Hello, I removed the problem with the index.js and also added the submission form for the blog and events. I added the translation and the library that allow us to send an email to the client once he subscribes, and we receive an email too. With the information entered inside the form, the client will receive the information he entered too, and he will also receive a welcome message. While subscribing, the client will have a captcha. Now I will add the firebase and fix what there is to fix.

Zino0031 reviewed (approved):
Review:

I'm really impressed with the way you've handled this pull request. You've not only addressed all the changes and feedback but also went above and beyond by adding a  translation, and email functionality. Your dedication to improving our project is commendable. I approve this. 👍🚀

mohamed70601 reviewed (approved):
All previous issues ironed out, good job!

DigitalDeveloperAM reviewed (approved):
Excellent work @melissasidisid .

The button is perfect and all language file are looking good.

This PR is good to go. Well done!
